### PR TITLE
fix pandas warnings in dataset generation

### PR DESCRIPTION
**Pull request by lostella**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-270/1 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-270/1/index.html

File: src/gluonts/dataset/repository/_lstnet.py [hunk 1]
```diff
@@ -174,16 +174,16 @@ def generate_lstnet_dataset(dataset_path: Path, dataset_name: str):
 
     # time of the first prediction
     prediction_dates = [
-        training_end + i * ds_info.prediction_length
+        training_end + (i * ds_info.prediction_length) * training_end.freq
```
Comment (jaheba) [inline src/gluonts/dataset/repository/_lstnet.py]: There is [`frequency_add`](https://github.com/awslabs/gluon-ts/blob/95201b0bbbafd7d8a618217e64a0cae6b6eaf025/src/gluonts/support/pandas.py#L18)

Maybe we can use it, or something similar instead?

Comment (lostella) [reply src/gluonts/dataset/repository/_lstnet.py]: true! forgot about that, fixing

**Comment by szha** [comment]
Job PR-270/2 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-270/2/index.html

**Pull request merged**